lib/cli/ui/progress.rb

```diff
@@ -72,7 +72,7 @@ def tick(percent: 0.01, set_percent: nil)
       # Format the progress bar to be printed to terminal
       #
       def to_s
-        suffix = " #{(@percent_done * 100).round(2)}%"
+        suffix = " #{(@percent_done * 100).floor}%"
```

Comment:
It occurred to me that this is way simpler than

- Padding (which adds unnecessary extra space)
- Having a consistent number of decimal places (unnecessary for round percentages)
- Adding logic to gradually ramp up precision if necessary (too complex)

Reply:
I like this better.  The extra precision really shouldn't be necessary.  But I do think padding to keep the bar a consistent size all the way up to 100 makes sense: 
```suggestion
        suffix = sprintf("%4i%%", (percent_done * 100).floor)
```

Reply:
This results in 

![right_justify_progress](https://user-images.githubusercontent.com/325821/48444311-95733700-e761-11e8-83d5-87ec5430e553.gif)

which I find better but has the downside of making the space between the progress bar and the percentage inconsistent.

If we were to left-align rather than right align we get

```suggestion
        suffix = " #{(@percent_done * 100).floor}%".ljust(5)
```

![left_justify_progress](https://user-images.githubusercontent.com/325821/48444546-4548a480-e762-11e8-87bb-baec825bbadf.gif)

Thoughts?

<details>
<summary>Test code</summary>

```ruby
$LOAD_PATH.unshift(File.join(__dir__, 'lib'))

require 'cli/ui'

CLI::UI::Progress.progress do |bar|
  (0..100).each do |percent|
    bar.tick(set_percent: percent / 100.0)
    sleep 0.05
  end
end
```

</details>

Reply:
🤷‍♀️ Your code, your call. Both look good to me, no strong opinion.

Reply:
Or ask a designer to do a user study with double blind